### DISCUSSION TITLE
Error at every 2nd reload

### DISCUSSION THREAD

**Discussion by marck-consulting** [LycheeOrg/Lychee#2522] [Bugs] [answered]
### Lychee version

5.5.1

### Which PHP version are you using?

PHP 8.3

### Detailed description of the problem

At every other reload or access to lychee I get these two errors:

> Refused to execute https://gallery.mapictures.de/livewire/livewire.min.js?id=c4fc8c5d as script because "X-Content-Type-Options: nosniff" was given and its Content-Type is not a script MIME type.

and

> ReferenceError: Can't find variable: Alpine

With these the page isn't working correctly and I just need to reload to get everything working as expected again until I press another button or reload again.

If needed, I could provide URL and user for testing purpose.

### Steps to reproduce the issue

Clean installation, livewire active, present at every other refresh or access of lychee.

### Diagnostics [REQUIRED]

 Diagnostics
    -----------
        Warning: Dropbox import not working. dropbox_key is empty.
        Warning: You may experience problems when uploading a photo of large size or handling many/large albums. Take a look in the FAQ for details.
        Warning: zend.assertions is enabled although Lychee is not in debug mode. Outside of debugging, code generation for assertions is recommended to be disabled for efficiency reasons
        Error: sys_get_temp_dir() is not readable, this will prevent you from uploading pictures.
    
    System Information
    ------------------
        Lychee Version (release):                5.5.1
        DB Version:                              5.5.1
        
        composer install:                        --no-dev
        APP_ENV:                                 production
        APP_DEBUG:                               false
        APP_URL:                                 set
        APP_DIR:                                 default
        LOG_VIEWER_ENABLED:                      true
        LIVEWIRE_ENABLED:                        false
        
        System:                                  Linux
        PHP Version:                             8.3.8-nmm1
        PHP User agent:                          Lychee/4 (https://lycheeorg.github.io/)
        Timezone:                                Europe/Berlin
        Max uploaded file size:                  384M
        Max post size:                           384M
        Livewire chunk size:                     12.00 MB
        Max execution time:                      60
        MySQL Version:                           10.5.25-MariaDB-ubu2004-log
        
        exec() Available:                        yes
        Imagick Available:                       1
        Imagick Enabled:                         1
        Imagick Version:                         1690
        GD Version:                              2.2.5
        Number of foreign key:                   12 found.
    
    Config Information
    ------------------
        version:                                 050501
        check_for_updates:                       1
        sorting_photos_col:                      taken_at
        sorting_photos_order:                    ASC
        sorting_albums_col:                      max_taken_at
        sorting_albums_order:                    ASC
        imagick:                                 1
        skip_duplicates:                         0
        small_max_width:                         0
        small_max_height:                        360
        medium_max_width:                        1920
        medium_max_height:                       1080
        lang:                                    de
        image_overlay_type:                      desc
        default_license:                         reserved
        compression_quality:                     90
        grants_full_photo_access:                1
        delete_imported:                         0
        mod_frame_enabled:                       1
        mod_frame_refresh:                       30
        thumb_2x:                                1
        small_2x:                                1
        medium_2x:                               1
        landing_page_enable:                     0
        force_migration_in_production:           0
        search_public:                           0
        hide_version_number:                     1
        SL_enable:                               0
        SL_for_admin:                            0
        recent_age:                              1
        grants_download:                         1
        photos_wraparound:                       1
        map_display:                             0
        zip64:                                   1
        map_display_public:                      0
        map_provider:                            Wikimedia
        force_32bit_ids:                         0
        map_include_subalbums:                   0
        update_check_every_days:                 3
        has_exiftool:                            1
        share_button_visible:                    0
        import_via_symlink:                      0
        has_ffmpeg:                              0
        location_decoding:                       0
        location_decoding_timeout:               30
        location_show:                           1
        location_show_public:                    0
        rss_enable:                              0
        rss_recent_days:                         7
        rss_max_items:                           100
        prefer_available_xmp_metadata:           0
        editor_enabled:                          1
        lossless_optimization:                   0
        swipe_tolerance_x:                       150
        swipe_tolerance_y:                       250
        local_takestamp_video_formats:           .avi|.mov
        log_max_num_line:                        1000
        unlock_password_photos_with_url_param:   0
        nsfw_visible:                            1
        nsfw_blur:                               0
        nsfw_warning:                            0
        nsfw_warning_admin:                      0
        nsfw_banner_override:                    
        map_display_direction:                   1
        album_subtitle_type:                     oldstyle
        upload_processing_limit:                 6
        new_photos_notification:                 0
        legacy_id_redirection:                   1
        zip_deflate_level:                       6
        default_album_protection:                1
        album_decoration:                        photo
        album_decoration_orientation:            row
        allow_username_change:                   0
        auto_fix_orientation:                    1
        use_job_queues:                          0
        random_album_id:                         starred
        use_last_modified_date_when_no_exif_date: 0
        layout:                                  justified
        date_format_photo_thumb:                 D d, F o - G:i:s
        date_format_photo_overlay:               D d, F o - G:i:s
        date_format_sidebar_uploaded:            D d, F o - G:i:s
        date_format_sidebar_taken_at:            D d, F o - G:i:s
        date_format_hero_min_max:                F o
        date_format_hero_created_at:             D d, F o - G:i:s
        date_format_album_thumb:                 F o
        upload_chunk_size:                       0
        nsfw_banner_blur_backdrop:               0
        search_pagination_limit:                 1000
        search_minimum_length_required:          4
        photo_layout_justified_row_height:       320
        photo_layout_masonry_column_width:       300
        photo_layout_grid_column_width:          250
        photo_layout_square_column_width:        200
        photo_layout_gap:                        12
        display_thumb_album_overlay:             always
        display_thumb_photo_overlay:             hover
        default_album_thumb_aspect_ratio:        1/1
        use_album_compact_header:                0
        login_button_position:                   left
        back_button_enabled:                     0
        back_button_text:                        Return to Home
        current_job_processing_visible:          1
        SA_random_thumbs:                        0
        job_processing_queue_visible:            1
        login_required:                          1
        enable_unsorted:                         0
        enable_starred:                          0
        enable_recent:                           0
        enable_on_this_day:                      0
        login_required_root_only:                1

### Browser & System [REQUIRED]

MacOS with Firefox, Chrome or Safari

### Please confirm (incomplete submissions will not be addressed)

- [X] I have provided easy and step-by-step instructions to reproduce the bug.
- [X] I understand my bug report will be removed if I haven't met the criteria above.

**Comment by ildyria** [comment, marked as answer]
Weird bug.

The bad news: I will not fix it (sorry).
The good news: It will be fixed with v6. Read more [here](https://lycheeorg.github.io/2024-06-29-future-of-lychee/).

**Reply by marck-consulting** [reply]
Agreed :D... but since it is probably not worthwhile to delve deeper into that (regarding the focus on v6 without livewire) I'm just gonna deactivate livewire and wait for v6 instead ;-).

Thanks for your quick feedback.
